synthtool/gcp/templates/node_split_library/index.ts.j2

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ** This file is automatically generated by synthtool. **
+// ** https://github.com/googleapis/synthtool **
+// ** All changes to this file may be overwritten. **
+
+{% for version in versions %}import * as {{ version }} from './{{ version}}';{{ "\n" }}{% endfor %}
+{% for client in clients %}const {{ client }} = {{ default_version }}.{{ client }};{{ "\n" }}{% endfor %}
+export {{ "{" }}{{ versions|join(', ')}}, {{ clients|join(', ')}}{{ "}" }};
```

Comment:
I think we would potentially want to be able to export a default client constructor too here, e.g.,

```
const SpeechClient = v1.SpeechClient;
export default {v1, v1beta1, SpeechClient};
```

Or else we end up always needing to reference an API version when creating an API client, which seems like it might lead to out of date docs throughout cloud.google.com.

I can have my mind changed on this issue, but it is a shift in how we've written Node.js clients.

Reply:
`clients` is extract from `{default_verison}/index.ts`, it could be a single client or multiple clients. So later on, if API producer adding more clients in default version, `src/index` looks like
```
const SpeechClient = v1.SpeechClient;
const SpeechClient2 = v1.SpeechClient2;
export default {v1, v1beta1, SpeechClient, SpeechClient2};
```
no need manual work.